src/ClassyResource/_makeRequest.js

```diff
@@ -32,7 +32,7 @@ export default function _makeRequest(path, method, headers = {}, form, data = {}
     : undefined;
 
   const requestParams = {
-    baseUrl: this.baseUrl,
+    baseUrl: baseUrl ? baseUrl : this.baseUrl,
```

Comment:
Where does `this.baseUrl` get set? I'd prefer that we update that value in the appropriate place if `config.oktaBaseUrl` is passed in.

Reply:
It is set in src/ClassyResource/main.js. How can we get input Request in ClassyResource to check is_okta_token?